_sass/blocks/how-it-works/_subpage_involved.scss

```diff
@@ -35,13 +42,27 @@
     @include omega-reset(2n);
     @include span-columns(3 of 12);
     @include omega(4n);
+  }
+}
+
+.revenues_subpage-involved_participate {
 
-    margin-top: $base-padding-large;
+  .container-half-space {
+    padding-right: $base-padding;
+  }
+
+  @include respond-to(medium-up) {
```

Comment:
Expected item on line 54 to appear before line 50. Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets